server/belga/io/feed_parsers/belga_newsml_1_2.py

```diff
@@ -739,19 +741,20 @@ def parse_descriptivemetadata(self, item, descript_el):
             for element in elements:
                 # country
                 if element.attrib.get('FormalName', '') == 'Country' and element.attrib.get('Value'):
-                    countries = get_resource_service('vocabularies').get_items(
-                        _id='countries',
-                        name=element.attrib.get('Value'),
-                        lang=item['language']
-                    )
-                    try:
-                        item.setdefault('subject', []).append(countries[0])
-                    except (StopIteration, IndexError) as e:
-                        logger.error(e)
+                    country_val = element.attrib.get('Value')
+                    countries = self._get_mapped_keywords(country_val, country_val.capitalize(), 'countries')
+                    if countries:
+                        item.setdefault('subject', []).extend(countries + self._get_country(countries[0]["qcode"]))
+
                 # city
                 if element.attrib.get('FormalName', '') == 'City' and element.attrib.get('Value'):
                     item.setdefault('extra', {})['city'] = element.attrib.get('Value')
 
+        # remove duplicated subject
+        item['subject'] = [
+            dict(i) for i, _ in itertools.groupby(sorted(item['subject'], key=lambda k: k['qcode']))
```

Comment:
I have checked it and found that it will work perfectly fine. here we are just sorting using qcode then groupby will automatically remove duplicate subjects.